Ontology/Willow/Asset/Asset.json

```diff
@@ -282,33 +270,6 @@
       "schema": "date",
       "writable": true
     },
-    {
```

Comment:
We need to keep IP Address, MAC Address, and modelNumber. Even if not every Asset twin defines these, they are fundamental to a basic Asset definition.